meson_options.txt

```diff
@@ -40,7 +40,7 @@ option('enable-fp16', type: 'boolean', value: false)
 option('enable-cublas', type: 'boolean', value: false)
 option('enable-openmp', type: 'boolean', value: true)
 option('enable-neon', type: 'boolean', value: false)
-option('enable-avx', type: 'boolean', value: false)
+option('enable-avx', type: 'boolean', value: true)
```

Comment:
For those who depend on arch, I recommend "feature" instead of "boolean".

If this is to be kept "boolean" with "true", meson should be able to check the arch and turn avx off if it's not x64.